networkit/cpp/graph/Graph.cpp

```diff
@@ -983,30 +983,48 @@ bool Graph::checkConsistency() const {
 /* ATTRIBUTE PREMISE AND INDEX CHECKS */
 
 template <>
-void Graph::ASB<Graph::PerNode>::checkPremise() const {
+void ASB<PerNode, Graph>::checkPremise() const {
```

Comment:
these four methods should be defined in a `Attributes.cpp` and not in `Graph.cpp`?